lib/cli/src/Cardano/CLI.hs

```diff
@@ -102,6 +109,27 @@ parseArgWith cli args option = do
     getArgOrExit :: Arguments -> Option -> IO String
     getArgOrExit = getArgOrExitWith cli
 
+parseAllArgsWith :: FromText a => Docopt -> Arguments -> Option -> IO [a]
+parseAllArgsWith cli args option = do
+    (mapM (fromText . T.pack) <$> args `getAllArgsOrExit` option) >>= \case
+        Right a -> do
+            return a
+        Left e -> do
+            putErrLn $ T.pack $ getTextDecodingError e
+            exitFailure
+  where
+    getAllArgsOrExit :: Arguments -> Option -> IO [String]
+    getAllArgsOrExit = getAllArgsOrExitWith cli
+
+-- | Same as 'getAllArgs', but 'exitWithUsage' if empty list.
+--
+--   As in 'getAllArgs', if your usage pattern required the option, 'getAllArgsOrExitWith' will not exit.
```

Comment:
https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters

Reply:
fixed

Reply:
hm - how did I forget this one :S